src/main/resources/js/moduleProvidedLibs.js

```diff
@@ -0,0 +1,132 @@
+//#endif
+
+// Libraries
+global.ChatLib = Java.type("com.chattriggers.ctjs.minecraft.libs.ChatLib");
+global.EventLib = Java.type("com.chattriggers.ctjs.minecraft.libs.EventLib");
+
+global.Renderer = Java.type("com.chattriggers.ctjs.minecraft.libs.renderer.Renderer");
+global.Shape = Java.type("com.chattriggers.ctjs.minecraft.libs.renderer.Shape");
+global.Rectangle = Java.type("com.chattriggers.ctjs.minecraft.libs.renderer.Rectangle");
+global.Text = Java.type("com.chattriggers.ctjs.minecraft.libs.renderer.Text");
+global.Image = Java.type("com.chattriggers.ctjs.minecraft.libs.renderer.Image");
+
+global.Tessellator = Java.type("com.chattriggers.ctjs.minecraft.libs.Tessellator");
+global.FileLib = Java.type("com.chattriggers.ctjs.minecraft.libs.FileLib");
+global.MathLib = Java.type("com.chattriggers.ctjs.minecraft.libs.MathLib");
+
+// Objects
+global.Display = Java.type("com.chattriggers.ctjs.engine.langs.js.JSDisplay");
+global.DisplayLine = Java.type("com.chattriggers.ctjs.engine.langs.js.JSDisplayLine");
+global.DisplayHandler = Java.type("com.chattriggers.ctjs.minecraft.objects.display.DisplayHandler");
+global.Gui = Java.type("com.chattriggers.ctjs.engine.langs.js.JSGui");
+global.Message = Java.type("com.chattriggers.ctjs.minecraft.objects.message.Message");
+global.TextComponent = Java.type("com.chattriggers.ctjs.minecraft.objects.message.TextComponent");
+global.Book = Java.type("com.chattriggers.ctjs.minecraft.objects.Book");
+global.KeyBind = Java.type("com.chattriggers.ctjs.minecraft.objects.KeyBind");
+global.Image = Java.type("com.chattriggers.ctjs.minecraft.libs.renderer.Image");
+global.Sound = Java.type("com.chattriggers.ctjs.minecraft.objects.Sound");
+global.PlayerMP = Java.type("com.chattriggers.ctjs.minecraft.wrappers.objects.PlayerMP");
+
+// Wrappers
+global.Client = Java.type("com.chattriggers.ctjs.minecraft.wrappers.Client");
+global.Player = Java.type("com.chattriggers.ctjs.minecraft.wrappers.Player");
+global.World = Java.type("com.chattriggers.ctjs.minecraft.wrappers.World");
+global.Server = Java.type("com.chattriggers.ctjs.minecraft.wrappers.Server");
+global.Inventory = Java.type("com.chattriggers.ctjs.minecraft.wrappers.objects.inventory.Inventory");
+global.TabList = Java.type("com.chattriggers.ctjs.minecraft.wrappers.TabList");
+global.Scoreboard = Java.type("com.chattriggers.ctjs.minecraft.wrappers.Scoreboard");
+global.CPS = Java.type("com.chattriggers.ctjs.minecraft.wrappers.CPS");
+global.Item = Java.type("com.chattriggers.ctjs.minecraft.wrappers.objects.inventory.Item");
+global.Block = Java.type("com.chattriggers.ctjs.minecraft.wrappers.objects.block.Block");
+global.Sign = Java.type("com.chattriggers.ctjs.minecraft.wrappers.objects.block.Sign");
+global.Entity = Java.type("com.chattriggers.ctjs.minecraft.wrappers.objects.Entity");
+global.Action = Java.type("com.chattriggers.ctjs.minecraft.wrappers.objects.inventory.action.Action");

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['Java' is not defined.](https://app.codacy.com/manual/FalseHonesty/ct.js/pullRequest?prid=5138651)